doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -107,6 +107,11 @@ nRF9160: Serial LTE Modem
   * Added native TLS support to the HTTPS client.
   * Added the #XCMNG command to support the use of native TLS.
 
+nRF Machine Learning (Edge Impulse)
+-----------------------------------
+
+* Increased value of :kconfig:option:`CONFIG_CAF_POWER_MANAGER_TIMEOUT` to 30 seconds on Thingy:53.
```

Comment:
```suggestion
* Increased the default value of :kconfig:option:`CONFIG_CAF_POWER_MANAGER_TIMEOUT` to 30 seconds for Thingy:53.
```

Reply:
Change a bit to:
`Increased the value of :kconfig:option:`CONFIG_CAF_POWER_MANAGER_TIMEOUT` to 30 seconds for Thingy:53.`
The Kconfig option value is set to 30 seconds in application configuration, we do not actually modify default value.